### ISSUE TITLE
Can i continue trainin add new classes?

### ISSUE THREAD

**Issue by AshleyRoth** [AlexeyAB/darknet#725]
Hello! I have already trained weights of 10 classes. And i need now add 2 classes. Can I add 2 more classes and continue training? (only 12 classes should work).

Will this work?

Thanks!

**Comment by cipri-tom** [comment]
No, this is currently a limitation of neural networks. You have to train from the start
